lang/cs/Org.Apache.REEF.Client.Tests/YarnClientTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@markusweimer We've had a similar discussion before. Should we formalize which type to use?